### DISCUSSION TITLE
Insert HTML near the getSelected()

### DISCUSSION THREAD

**Discussion by JGalbois** [GrapesJS/grapesjs#4421] [General]
Hi,
Sorry for my english.

I want to insert HTML just before, or just after the selected element.
But today, I find just one solution : 
`editor.getSelected().parent().append('<label>Azerty</label>');`

But not perfect
Have you got an other solution ?
I want to insert a label before an input, or a price's span after an input

Thank's

J

**Comment by artf** [comment]
With the `append` method you can specify the `at` option as an index
```js
cmp.parent().append('...', { at: cmp.index() })
```

**Reply by JGalbois** [reply]
Great
Thank's a lot
